examples/trace/src/main/java/com/google/cloud/opentelemetry/example/trace/TraceExporterExample.java

```diff
@@ -17,23 +17,23 @@
 
 import com.google.cloud.opentelemetry.trace.TraceConfiguration;
 import com.google.cloud.opentelemetry.trace.TraceExporter;
-import io.opentelemetry.api.GlobalOpenTelemetry;
 import io.opentelemetry.api.trace.Span;
-import io.opentelemetry.api.trace.Tracer;
 import io.opentelemetry.context.Scope;
 import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.common.CompletableResultCode;
 import io.opentelemetry.sdk.trace.SdkTracerProvider;
 import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
 import io.opentelemetry.sdk.trace.export.SpanExporter;
 import java.time.Duration;
 import java.util.Random;
+import java.util.concurrent.TimeUnit;
 
 public class TraceExporterExample {
-  // TODO: Update this to NON-global OpenTelemetry.  Only APIs/Frameworks should be using global.
-  private static final Tracer tracer =
-      GlobalOpenTelemetry.getTracer("io.opentelemetry.example.TraceExporterExample");
+  private static final String INSTRUMENTATION_SCOPE_NAME = TraceExporter.class.getName();
```

Comment:
NIt: This should be the TraceExporterExample, to match what was there previously.